app/renderer/components/navigation/urlBarSuggestions.js

```diff
@@ -36,12 +38,12 @@ class UrlBarSuggestions extends ImmutableComponent {
 
   render () {
     const suggestions = this.props.suggestionList
-    const bookmarkSuggestions = suggestions.filter((s) => s.type === suggestionTypes.BOOKMARK)
-    const historySuggestions = suggestions.filter((s) => s.type === suggestionTypes.HISTORY)
-    const aboutPagesSuggestions = suggestions.filter((s) => s.type === suggestionTypes.ABOUT_PAGES)
-    const tabSuggestions = suggestions.filter((s) => s.type === suggestionTypes.TAB)
-    const searchSuggestions = suggestions.filter((s) => s.type === suggestionTypes.SEARCH)
-    const topSiteSuggestions = suggestions.filter((s) => s.type === suggestionTypes.TOP_SITE)
+    const bookmarkSuggestions = suggestions.filter((s) => s.get('type') === suggestionTypes.BOOKMARK)
```

Comment:
That would be better but not introduced in this commit so leaving as is for now. The right way is to store it separately in state so no filter is needed at all.